### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by pixfan**

### CODE REVIEW

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855

**Comment by pixfan** [comment]
Devs funciona, pasamos a staging

**Comment by pixfan** [comment]
## Model metrics

Training accuracy: 1.0%

Mejores parámetros: {'classifier\_\_learning_rate': 0.97, 'classifier\_\_n_estimators': 2}
Mejor Accuracy en CV:0.9945419103313841
precision    recall  f1-score   support

           0       0.98      1.00      0.99       193
           1       1.00      0.99      1.00       662

    accuracy                           1.00       855

macro avg       0.99      1.00      0.99       855
weighted avg       1.00      1.00      1.00       855
